chapter_attention-mechanisms/nadaraya-waston.md

```diff
@@ -93,7 +93,7 @@ n_test
 def plot_kernel_reg(y_hat):
     d2l.plot(x_test, [y_truth, y_hat], 'x', 'y', legend=['Truth', 'Pred'],
              xlim=[0, 5], ylim=[-1, 5])
-    d2l.plt.plot(x_train, y_train, 'o', alpha=0.5);
+    d2l.plt.plot(x_train, y_train, 'o', alpha=0.5)
```

Comment:
we may still need this `;`, or the plot may not render... can you add it back?

Reply:
OK